css/style.css

```diff
@@ -16,4 +15,118 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+header {
+  background-color: white;
+}
+.image-content {
+  background-image: url("../img/first-background.jpg");
+  background-size: cover;
+  padding: 15rem;
+  color: white;
+  text-align: center;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 100%;
+}
+
+.image-content p {
+  font-size: 20px;
+}
+
+/* Nav styling starts here */
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 2px;
+}
+
+.nav-links {
+  display: flex;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav-links a {
+  color: gray;
+  padding: 10px 20px;
+  text-decoration: none;
+}
+
+/* section styling starts here */
+.how-container {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+}
 
+.how_it_works {
```

Comment:
Some of your class names use underscores while the others use dashes. I'd stick to dashes as that's more natural to CSS.